File: src/at/fhooe/mc/emg/client/ClientDataCallback.java
package at.fhooe.mc.emg.client;

/**
 * Author:  Martin Macheiner
 * Date:    03.07.2017
 */
public interface ClientDataCallback {

    void onChanneledDataAvailable(ChannelData channelData);

    void onRawDataAvailable(String line);
}

File: client/src/at/fhooe/mc/emg/client/ClientDataCallback.kt
package at.fhooe.mc.emg.client

/**
 * Author:  Martin Macheiner
 * Date:    03.07.2017
 */
interface ClientDataCallback {

    fun onChanneledDataAvailable(channelData: ChannelData)

    fun onRawDataAvailable(line: String)
}